website/docs/zombie-flags.mdx

```diff
@@ -0,0 +1,107 @@
+---
+id: zombie-flags
+title: Zombie Flags
+description: This page explains how to identify and manage Zombie Flags in ConfigCat through the dashboard, email reports, or the Zombie Flags API.
+---
+
+Zombie flags are feature flags that are no longer needed or actively used. They often remain in the codebase long after their purpose has been fulfilled. Removing these outdated flags can reduce technical debt, improve performance, and simplify your codebase.
+
+Common scenarios where zombie flags appear:
+
+- A temporary feature becomes permanent, but its flag is never removed.
+- A flag was introduced for a one-time rollout or experiment that's now complete.
+- A decision was made to always keep a feature on or off, making the flag redundant.
+- The flag hasn't been toggled or referenced for an extended period.
+
+## Why Removing Zombie Flags Matters
+
+Leaving stale flags in your code can lead to the following:
+- **Increased complexity:** Unused flags clutter code and reduce maintainability.
+- **Unexpected behavior:** Outdated flags might trigger legacy logic unintentionally.
+- **Longer onboarding and reviews:** Developers waste time reviewing obsolete logic.
+- **Performance degradation:** In large-scale systems, excess flags can slow down operations.
+
+## Managing Zombie Flags in ConfigCat
+
+There are three ways to access and manage Zombie flags in ConfigCat:
+
+- Through the [Dashboard](https://app.configcat.com/product/zombie-flags).
+- Using the [Zombie Flags Email Report](https://app.configcat.com/my-account/zombie-flags-report).
+- Via the [Zombie Flags API](https://api.configcat.com/docs/index.html#tag/Zombie-(stale)-flags).
+
+### Using the Dashboard
+
+Using the Dashboard is the quickest way to view your zombie flags. Here's how to access the Zombie Flags page:
+
+1. In the [organization overview page](https://app.configcat.com/organization), select the product.
```

Comment:
I think we should only give the https://app.configcat.com/product/zombie-flags url here without this step by step guide. And i think the first screenshot is not necessary.

I'd remove the "Here's how to access the Zombie Flags page:" part and the step by step and simply make the Zombie flags page a link in the "The Zombie flags page shows all the stale flags in a table for the selected product based on the filters set:" sentence.